libraries/AP_DDS/AP_DDS_Service_Table.h

```diff
@@ -51,6 +55,28 @@ constexpr struct AP_DDS_Client::Service_table AP_DDS_Client::services[] = {
         .reply_topic_name = "rr/ap/mode_switchReply",
     },
 #endif // AP_DDS_MODE_SWITCH_SERVER_ENABLED
+#if AP_DDS_RALLY_SERVER_ENABLED
+    {
+        .req_id = to_underlying(ServiceIndex::GET_RALLY),
+        .rep_id = to_underlying(ServiceIndex::GET_RALLY),
+        .service_rr = Service_rr::Replier,
+        .service_name = "rs/ap/rally_getService",
+        .request_type = "ardupilot_msgs::srv::dds_::RallyGet_Request_",
+        .reply_type = "ardupilot_msgs::srv::dds_::RallyGet_Response_",
+        .request_topic_name = "rq/ap/rally_getRequest",
+        .reply_topic_name = "rr/ap/rally_getReply",
+    },
+    {
+        .req_id = to_underlying(ServiceIndex::SET_RALLY),
+        .rep_id = to_underlying(ServiceIndex::SET_RALLY),
+        .service_rr = Service_rr::Replier,
+        .service_name = "rs/ap/rally_setService",
+        .request_type = "ardupilot_msgs::srv::dds_::RallySet_Request_",
```

Comment:
It's probably a good idea to construct a string like this from "ardupilot_msgs::srv::dds_::" + the request - this will save a significant amount of flash because of how many times this substring seems to be used. 
You might also want to do the same thing with "rs/ap/rally_"

Reply:
that seems like a good idea. I can apply that to both service and topic file.

Reply:
Those are not stings, so it would make the code pretty unreadable. I would push this optimization to a dedicated PR.